cpp/ql/src/Likely Bugs/OO/UnsafeUseOfThis.ql

```diff
@@ -12,8 +12,11 @@
  *       security
  */
 
-import semmle.code.cpp.ir.IR
 import cpp
+// We don't actually use the global value numbering library in this query, but without it we end up
+// recomputing the IR.
+private import semmle.code.cpp.valuenumbering.GlobalValueNumbering
```

Comment:
If this is true shouldn't `semmle.code.cpp.valuenumbering.GlobalValueNumbering` be imported in `semmle.code.cpp.ir.IR`, so that all users benefit?

Reply:
It was indeed true: 
![time_split_absolute](https://user-images.githubusercontent.com/4527323/100194171-7e8d8080-2ef5-11eb-9e52-615c4d9969de.png)
As far as I know, this cache re-evaluation problem happens only in unlucky cases. We can maybe this at the team meeting on Thursday? In any case, I don't think we should do that change in this PR.